community/dbms/src/test/java/org/neo4j/commandline/dbms/ImportCommandTest.java

```diff
@@ -198,67 +198,83 @@ public void shouldPrintNiceHelp() throws Throwable
+                            "      one import, where each data source has its own header. Note that file\n" +
+                            "      groups must be enclosed in quotation marks. [default:]\n" +
+                            "  --id-type=<STRING|INTEGER|ACTUAL>\n" +
+                            "      Each node must provide a unique id. This is used to find the correct nodes\n" +
+                            "      when creating relationships. Possible values are:\n" +
+                            "        STRING: arbitrary strings for identifying nodes,\n" +
+                            "        INTEGER: arbitrary integer values for identifying nodes,\n" +
+                            "        ACTUAL: (advanced) actual node ids.\n" +
+                            "      For more information on id handling, please see the Neo4j Manual:\n" +
+                            "      https://neo4j.com/docs/operations-manual/current/tools/import/\n" +
+                            "      [default:STRING]\n" +
+                            "  --input-encoding=<character-set>\n" +
+                            "      Character set that input data is encoded in. [default:UTF-8]\n" +
+                            "  --ignore-extra-columns=<true|false>\n" +
+                            "      If un-specified columns should be ignored during the import.\n" +
+                            "      [default:false]\n" +
+                            "  --ignore-duplicate-nodes=<true|false>\n" +
+                            "      If duplicate nodes should be ignored during the import. [default:false]\n" +
+                            "  --ignore-missing-nodes=<true|false>\n" +
+                            "      If relationships referring to missing nodes should be ignored during the\n" +
+                            "      import. [default:false]\n" +
+                            "  --multiline-fields=<true|false>\n" +
+                            "      Whether or not fields from input source can span multiple lines, i.e.\n" +
+                            "      contain newline characters. [default:false]\n" +
+                            "  --delimiter=<,>\n" +
+                            "      Delimiter character between values in CSV data. [default:,]\n" +
+                            "  --array-delimiter=<,>\n" +
+                            "      Delimiter character between array elements within a value in CSV data.\n" +
+                            "      [default:;]\n" +
+                            "  --max-memory=<max-memory-that-importer-can-use>\n" +
+                            "      Maximum memory that neo4j-admin can use for various data structures and\n" +
+                            "      caching to improve performance. Values can be plain numbers, like 10000000\n" +
+                            "      or e.g. 20G for 20 gigabyte, or even e.g. 70%. [default:90%]\n",
```

Comment:
This test will fail on Windows since you are using `\n` lineendings. Note that the previous version used `%n` together with `String.format` to get a platform-dependant lineending.